rca/events/models.py

```diff
@@ -618,6 +621,75 @@ def get_booking_bar(self):
             ),
         }
 
+    def add_slashes(self, string):
```

Comment:
Suggestion: this utility function could do with a more descriptive name, eg `ical_escape_characters` and a comment.

```
"""
Escapes special characters in long form text fields for iCal records.
"""
```